gnutls/gnutls-3.6.3-msysize.patch

```diff
@@ -12,7 +12,7 @@ diff -Naur gnutls-3.6.3-orig/build-aux/ar-lib gnutls-3.6.3/build-aux/ar-lib
  	    file_conv=wine
  	    ;;
 @@ -68,6 +71,9 @@
- 	cygwin)
+ 	cygwin | msys)
  	  file=`cygpath -m "$file" || echo "$file"`
  	  ;;
 +	msys)
```

Comment:
And this one

Reply:
Ya, I realized that after pushing that. But this may not cause severe issue. I shall remove it next update.